src/views/AnnualReport.vue

```diff
@@ -325,9 +325,8 @@ import {
 // Mixins
 import { DateMixin, FilingMixin, ResourceLookupMixin, BcolMixin } from '@/mixins'
 
-// Enums, Constants and Interfaces
-import { FilingCodes, FilingStatus, FilingTypes, StaffPaymentOptions } from '@/enums'
-import { APPOINTED, CEASED, NAMECHANGED, ADDRESSCHANGED, DASHBOARD } from '@/constants'
```

Comment:
Here's a good example of why global constants are a bad idea -- which of these is to be used for what !?!?

This is why I changed them to 2 new enums.